.github/workflows/busted.yml

```diff
@@ -1,46 +1,62 @@
+# ATTENTION
+#
+# This is *not* a typical busted workflow! Do not copy this to other projects!
+# Instead of installing busted from somewhere and using it on a code base,
+# we're actually building and running the current HEAD version on ourselves:
+# basically this is a combination bootstrap test and dogfooding. The upshot is
+# that we check both that everything works (and end-to-end test) and that the
+# current codebase checks against itself. This ensures we can fix a bug or make
+# a breaking change in busted without being blocked by our own CI using a
+# different busted verision.
+#
+# See the README.md file for examples suitable for use in other projects.
+
 name: Busted
 
-on: [push, pull_request]
+on: [ push, pull_request ]
 
 jobs:
 
   busted:
     strategy:
       fail-fast: false
       matrix:
-        luaVersion: ["5.4", "5.3", "5.2", "5.1", "luajit", "luajit-openresty"]
-        penlightVersion: ["1.9.2", "1.8.0", "1.6.0", "1.3.2"]
-    runs-on: ubuntu-20.04
+        luaVersion: [ "5.4", "5.3", "5.2", "5.1", "luajit" ] #, "luajit-openresty" ]
```

Comment:
any reason to comment out the openresty version?

Reply:
Yes. Something is wrong with the @leafo's openresty image. It regularly but seemingly randomly crashes (maybe once every ten runs on small projects up to almost 100% on some specific large ones). There is some segfault problem that doesn't happen with openresty locally only in the GH runner images. Many other projects have temporarily disabled this too. We had to do so recently for luasocket because it was crashing there and blocking merges. Disabling it here is preemptive but I think also reasonable given the number of false positive test failures it generates elsewhere.

Reply:
ok, that makes sense then